examples/build_dict.py

```diff
@@ -18,7 +18,7 @@ def build_dict(opt):
         return
     print('[ setting up dictionary. ]')
     if os.path.isfile(opt['dict_file']):
-        # dict already built
```

Comment:
maybe we should changing this to the full word since we are not referring to a python dict?